test/integration/CoreRegistration.t.sol

```diff
@@ -97,7 +97,7 @@ contract Test_CoreRegistration is MockAVSDeployer {
         cheats.prank(operator);
         delegationManager.registerAsOperator(
```

Comment:
This interface is different?

Reply:
yeah it changed for the deprecated value for the new rewards release

Reply:
Yeah I remember this was to avoid confusion with RewardsCoordinator earners

Reply:
@ypatil12 need an approval here if you're good with the resolution here